hack/make-rules/test.sh

```diff
@@ -292,6 +295,10 @@ runTests() {
   # command, which is much faster.
   if [[ ! ${KUBE_COVER} =~ ^[yY]$ ]]; then
     kube::log::status "Running tests without code coverage"
+    # shellcheck disable=SC2086
+    #   SC2086: Double quote to prevent globbing and word splitting
+    # It is detected at ${KUBE_RACE} and ${KUBE_TIMEOUT}. These variables might be
+    # empty or contain multiple words, should be split.
```

Comment:
shellcheck said SC2086.
```
In ./hack/make-rules/test.sh line 296:
      ${KUBE_RACE} ${KUBE_TIMEOUT} "${@}" \
      ^----------^ SC2086: Double quote to prevent globbing and word splitting.
                   ^-------------^ SC2086: Double quote to prevent globbing and word splitting.
```

If adding double quotes, `go test` could not work properly because recognized as the below.
I thought disabling SC2086 at this line was reasonable.

- with double quotes
```
go test '' '-timeout 120s' k8s.io/kubernetes/cluster/gce/cos k8s.io/kubernetes/cluster/gce/custom ... snip ...
        ^^^^^^^^^^^^^^^^^^
```

- without double quotes
```
go test -timeout 120s k8s.io/kubernetes/cluster/gce/cos k8s.io/kubernetes/cluster/gce/custom ... snip ...
```

Reply:
please do not use shellcheck disable comments.

this can be solved using the pattern we used for goflags

Reply:
I beleive KUBE_RACE is only ever one flag, so an even simpler pattern can be written to expand to null when unset.

take a look at https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html

Reply:
Thank you comment! I change the part to add these parameters into goflags array. Should I use new array?
Would you review again?